LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -1976,6 +1976,229 @@ TEMPLAR_PISTOL_RANGE[23] = -80
++SKIRMISHER_LASER_RANGE[4] = 15
++SKIRMISHER_LASER_RANGE[5] = 10
++SKIRMISHER_LASER_RANGE[6] = 5
++SKIRMISHER_LASER_RANGE[7] = 0
++SKIRMISHER_LASER_RANGE[8] = 0
++SKIRMISHER_LASER_RANGE[9] = 0
++SKIRMISHER_LASER_RANGE[10] = 0
++SKIRMISHER_LASER_RANGE[11] = 0
++SKIRMISHER_LASER_RANGE[12] = -4
++SKIRMISHER_LASER_RANGE[13] = -8
++SKIRMISHER_LASER_RANGE[14] = -12
++SKIRMISHER_LASER_RANGE[15] = -16
++SKIRMISHER_LASER_RANGE[16] = -20
++SKIRMISHER_LASER_RANGE[17] = -24
++SKIRMISHER_LASER_RANGE[18] = -30
++SKIRMISHER_LASER_RANGE[19] = -40
++SKIRMISHER_LASER_RANGE[20] = -50
++SKIRMISHER_LASER_RANGE[21] = -60
++SKIRMISHER_LASER_RANGE[22] = -70
++SKIRMISHER_LASER_RANGE[23] = -80
++SKIRMISHER_LASER_RANGE[24] = -90
++SKIRMISHER_LASER_RANGE[25] = -100
+
++BULLPUP_COIL_BASEDAMAGE=	(Damage=7,	Spread=2, PlusOne=50,	Crit=4, Pierce=1, Shred=0, Tag="", DamageType="Projectile_MagXCom")
+
++BULLPUP_COIL_AIM = 0
++BULLPUP_COIL_CRITCHANCE = 0
++BULLPUP_COIL_ICLIPSIZE = 3
++BULLPUP_COIL_ISOUNDRANGE=30
++BULLPUP_COIL_IENVIRONMENTDAMAGE = 0
++BULLPUP_COIL_UPGRADESLOTS=3;
+
++SKIRMISHER_COIL_RANGE[0] = 30
++SKIRMISHER_COIL_RANGE[1] = 30
++SKIRMISHER_COIL_RANGE[2] = 25
++SKIRMISHER_COIL_RANGE[3] = 20
++SKIRMISHER_COIL_RANGE[4] = 15
++SKIRMISHER_COIL_RANGE[5] = 10
++SKIRMISHER_COIL_RANGE[6] = 5
++SKIRMISHER_COIL_RANGE[7] = 0
++SKIRMISHER_COIL_RANGE[8] = 0
++SKIRMISHER_COIL_RANGE[9] = 0
++SKIRMISHER_COIL_RANGE[10] = 0
++SKIRMISHER_COIL_RANGE[11] = 0
++SKIRMISHER_COIL_RANGE[12] = -4
++SKIRMISHER_COIL_RANGE[13] = -8
++SKIRMISHER_COIL_RANGE[14] = -12
++SKIRMISHER_COIL_RANGE[15] = -16
++SKIRMISHER_COIL_RANGE[16] = -20
++SKIRMISHER_COIL_RANGE[17] = -24
++SKIRMISHER_COIL_RANGE[18] = -30
++SKIRMISHER_COIL_RANGE[19] = -40
++SKIRMISHER_COIL_RANGE[20] = -50
++SKIRMISHER_COIL_RANGE[21] = -60
++SKIRMISHER_COIL_RANGE[22] = -70
++SKIRMISHER_COIL_RANGE[23] = -80
++SKIRMISHER_COIL_RANGE[24] = -90
++SKIRMISHER_COIL_RANGE[25] = -100
+
+
++VEKTOR_LASER_BASEDAMAGE=	(Damage=4, Spread=1, PlusOne=75, Crit=4, Pierce=0, Shred=0, Tag="", DamageType="Projectile_BeamXCom")
++Vektor_Laser_ImagePath="CrossbowVektor.Inv_CrossbowVektor"
++VEKTORLASER_AIM = 5
++VEKTOR_LASER_CRITCHANCE = 0
++VEKTOR_LASER_ICLIPSIZE = 3
++VEKTOR_LASER_ISOUNDRANGE=30
++VEKTOR_LASER_IENVIRONMENTDAMAGE = 0
++VEKTOR_LASER_UPGRADESLOTS=3
+
++VEKTOR_LASER_RANGE[0]=0
++VEKTOR_LASER_RANGE[1]=25
++VEKTOR_LASER_RANGE[2]=24
++VEKTOR_LASER_RANGE[3]=22
++VEKTOR_LASER_RANGE[4]=21
++VEKTOR_LASER_RANGE[5]=18
++VEKTOR_LASER_RANGE[6]=16
++VEKTOR_LASER_RANGE[7]=14
++VEKTOR_LASER_RANGE[8]=12
++VEKTOR_LASER_RANGE[9]=10
++VEKTOR_LASER_RANGE[10]=9
++VEKTOR_LASER_RANGE[11]=8
++VEKTOR_LASER_RANGE[12]=7
++VEKTOR_LASER_RANGE[13]=6
++VEKTOR_LASER_RANGE[14]=4
++VEKTOR_LASER_RANGE[15]=2
++VEKTOR_LASER_RANGE[16]=1
++VEKTOR_LASER_RANGE[17]=0
++VEKTOR_LASER_RANGE[18]=0
++VEKTOR_LASER_RANGE[19]=0
++VEKTOR_LASER_RANGE[20]=0
++VEKTOR_LASER_RANGE[21]=0
++VEKTOR_LASER_RANGE[22]=0
++VEKTOR_LASER_RANGE[23]=0
++VEKTOR_LASER_RANGE[24]=0
++VEKTOR_LASER_RANGE[25]=0
```

Comment:
I think the SR's only got aim penalities because of squadsight, max non-squadsight LOS is 17. Regardless, I wanted to rework the reaper range tables anyways, with giving them penalty on close, and bonus on longer range.